runtime/lua/vim/treesitter/dev.lua

```diff
@@ -304,7 +314,7 @@ function M.inspect_tree(opts)
 
   local buf = api.nvim_get_current_buf()
   local win = api.nvim_get_current_win()
-  local pg = assert(TSTreeView:new(buf, opts.lang))
+  local treeview = assert(TSTreeView:new(buf, opts.lang))
```

Comment:
I went ahead and also renamed `pg` to `treeview` in this PR. The `pg` identifier is a holdover from the original implementation when this was still called the "playground". Since we no longer use that terminology the `pg` identifier is just confusing.